public/app/plugins/panel/table2/TablePanel.tsx

```diff
@@ -13,13 +12,41 @@ export class TablePanel extends Component<Props> {
     super(props);
   }
 
+  onColumnResize = (field: Field, width: number) => {
+    const current = this.props.fieldConfig;
+    const matcherId = FieldMatcherID.byName;
+    const prop = 'width';
+    const overrides = current.overrides.filter(
+      o => o.matcher.id !== matcherId || o.matcher.options !== field.name || o.properties[0].prop !== prop
+    );
+
+    overrides.push({
+      matcher: { id: matcherId, options: field.name },
+      properties: [{ custom: true, prop, value: width }],
+    });
+
+    this.props.onFieldConfigChange({
```

Comment:
This call blows up React first time, I think I need help understanding how to prevent this.

Reply:
Leaving this for now to be solved later.